### ISSUE TITLE
When a file does not expire, expiry date should be NULL not today date

### ISSUE THREAD

**Issue by lenamtl** [projectsend/projectsend#194]
When a file does not expire, expiry date should be NULL not today date.
I know there is a value for expire 0 or 1, but I think this is no good idea to set today date like this.

**Comment by ignacionelson** [comment]
How about hiding/showing the date field according to the checkbox state?

**Comment by crashff00** [comment]
I agree with lenamtl. I think that if the checkbox is empty, the expiry date should be = NULL to prevent any accidental expiration from code conflicts

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
